src/mixins/filing-template-mixin.ts

```diff
@@ -959,4 +977,29 @@ export default class FilingTemplateMixin extends DateMixin {
     }
     return toReturn
   }
+
+  /**
+   * Ensure consistent object structure for a Restoration, whether it contains a Name
+   * Request or not, and whether it is an initial draft or it has been previously saved.
+   * Object merging does not work very well otherwise due to nested properties.
+   * @param filing the filing fetched from legal-api
+   * @returns the filing in safe-empty state if applicable
+   */
+  formatEmptyRestoration (filing: any): RestorationFilingIF {
+    const toReturn = filing
+    if (toReturn.restoration) {
+      // set offices
+      if (!toReturn.restoration?.offices) {
+        toReturn.restoration.offices = []
```

Comment:
Is this still needed? If so then you should assign an object here, not an array.